Homework-8/task_3.py

```diff
@@ -0,0 +1,10 @@
+from datetime import datetime
+
+date = input("Enter the date in format YYYY-mm-ddTHH:MM:SS.ffffff+zz:zz\n")
+try:
+    date = datetime.strptime(date, "%Y-%m-%dT%H:%M:%S.%f%z")
+except ValueError:
+    print("Invalid date format")
+    exit(1)
+date = date.strftime("%d/%m/%Y %H:%M %z")
+print(date)
```

Comment:
ცოტა ცდება დავალებას. რომ ჩავწერე 2024-03-22T19:17:42.956376+00:00, დამიწერა 22/03/2024 19:17 +0000, როცა უნდა დაეწერა 22-03-2024 7:17:42 +0
მიხარია, რომ try except-ში გიწერია ფორმატირების ამბავი. 
გთავაზობ ალტერნატივას უშუალოდ ჩვენს ნასწავლზე დაყრნდნობით და ბიბლიოთეკის გარეშე, თუმცა იმას არ ვამბობ, რომ ბიბლიოთეკის გამოყენება ცუდია ან რაიმე მსგავსი:
date = input('Please enter date - ').strip()

day, rest = date.split('T')
day = '-'.join(day.split('-')[::-1])
time, zone = rest.split('.')
time = time.split(':')
time[0] = int(time[0])
if time[0] > 12:
    time[0] = time[0] - 12
time[0] = str(time[0])
time = ':'.join(time)

for sign in '-+':
    if sign in zone:
        zone = zone.split(sign)[1]
        break
zone = sign + str(int(zone.split(':')[0])) + (':' + zone.split(':')[1]).replace(':00','')

print(day, time, zone)